Ventivader/Ventivader/ViewModels/ConnectingBLEViewModel.swift

```diff
@@ -0,0 +1,20 @@
+////
```

Comment:
can we delete this file? or do you plan to refactor to a view model?

Reply:
Deleted this and the VC